### ISSUE TITLE
Reader: Table being read as paragraphs and lists instead of a table

### ISSUE THREAD

**Issue by alisan16** [department-of-veterans-affairs/caseflow#12244]
The following occurs with the Reader application: Data displayed visually on the page beginning with the column headers: Categories, Receipt Date, Document Type, Issue Tags and Comments should be coded as a table for assistive technology users. This text content is read as paragraph text and links. Assistive technology users cannot understand the information and structure that is presented visually. (New: 08/27/2019) the filter button would not work as well next to categories and the table is not coded as a table

<img width="1174" alt="Screen Shot 2019-09-27 at 5 14 15 PM" src="https://user-images.githubusercontent.com/1198106/65802753-9418e600-e14a-11e9-8811-d4a16f524f4a.png">

**Comment by yoomlam** [comment]
Difference compared to #12136? See comments on that ticket.
